lib/membrane_audio_mixer_bin.ex

```diff
@@ -90,123 +63,123 @@ defmodule Membrane.AudioMixerBin do
     availability: :always,
     caps: Raw
 
+  defmodule MixerOptions do
+    @moduledoc """
+    Structure representing options that would be passed to each created Membrane.AudioMixer element.
+    """
+    defstruct caps: nil, frames_per_buffer: 2048, prevent_clipping: true
+
+    @type t :: %__MODULE__{
+            caps: Raw.t(),
+            frames_per_buffer: pos_integer(),
+            prevent_clipping: boolean()
+          }
+  end
+
   @impl true
   def handle_init(options) do
     state =
       options
       |> Map.from_struct()
-      |> Map.put(:added_pad_count, 0)
-
-    {children, links} = create_mixers_tree(state)
-    spec = %ParentSpec{children: children, links: links}
+      |> Map.put(:inputs, 0)
+      |> Map.update!(:mixer_options, fn val ->
+        if val == nil, do: %MixerOptions{}, else: val
+      end)
```

Comment:
1. Anonymous functions support pattern matching clauses:
```suggestion
      |> Map.update!(:mixer_options, fn 
        nil -> %MixerOptions{}
        val -> val
      end)
```
2. Instead of updating the value if `nil`, you can just make it a default for an option